runelite-client/src/main/java/net/runelite/client/config/ConfigItem.java

```diff
@@ -46,4 +46,8 @@
 	String warning() default "";
 
 	boolean secret() default false;
+
+	int maxInt() default Integer.MAX_VALUE;
```

Comment:
I think you should instead use custom type as return value (e.g something like Range) instead of having maxInt and minInt available for all fields (e.g even for string fields and boolean fields where it makes no sense)